app/assets/javascripts/application.js

```diff
@@ -10,7 +10,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery
+= require jquery
```

Comment:
So funny story. It did break. Added back in the require and then it ran the javascript that WAS breaking the site. Fixed that indentation error and undid my un-require.